src/main/java/org/seqdoop/hadoop_bam/util/FileFormatUtils.java

```diff
@@ -0,0 +1,59 @@
+package org.seqdoop.hadoop_bam.util;
+
+import com.google.common.collect.ImmutableList;
+import htsjdk.samtools.util.Interval;
+import org.apache.hadoop.conf.Configuration;
+import org.seqdoop.hadoop_bam.FormatException;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.Supplier;
+
+/**
+ * Common utilities across different file formats.
+ */
+public abstract class FileFormatUtils {
```

Comment:
The class name seems overly generic, it's really just for interval parsing, so can you call it `IntervalUtil`?

Reply:
Happy to change it to whatever unless it creates collisions. I chose FileFormatUtils as it seems to be the "common denominator" of the two classes  the code is extracted from.